x11-apps/copyq/metadata.xml

```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+	<maintainer type="person">
+			<email>onigino@protonmail.com</email>
+			<name>Gino</name>
```

Comment:
For the record: this isn't a problem right now but it's likely that in the near future we will require all contributors to identify themselves by their legal name.

Reply:
Ok, I have had spam problems in the past, hence the special email address just for commits, etc.